specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/applications.json

```diff
@@ -0,0 +1,403 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HDInsightManagementClient",
+        "description": "The HDInsight Management Client.",
+        "version": "2015-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/applications": {
+            "get": {
+                "tags": [
+                    "Applications"
+                ],
+                "description": "Lists all of the applications HDInsight cluster.",
+                "operationId": "Applications_List",
+                "x-ms-examples": {
+                    "Get All Applications for an HDInsight cluster": {
+                        "$ref": "./examples/GetAllApplications.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {

```

Comment:
please consider adding `default:` response to each endpoint that will describe error response message structures. [https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#default-response](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#default-response) . You can take a look to some examples in other swaggers e.g. [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json#L68](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json#L68)

Reply:
This looks helpful. I'll look into this.